dali/operators/geometry/affine_transforms/transform_base_op.h

```diff
@@ -72,7 +72,7 @@ class TransformBaseOp : public SequenceOperator<Backend, true> {
 
   bool SetupImpl(std::vector<OutputDesc> &output_descs, const Workspace &ws) override {
     has_input_ = ws.NumInput() > 0;
-    auto curr_batch_size = has_input_ ? ws.GetInputBatchSize(0) : ws.GetRequestedBatchSize(0);
+    auto curr_batch_size = InferBatchSizeFromInput(ws);
```

Comment:
This defeats the purpose of RequestedBatchSize. Instead, move `InferBatchSize` to the executor and invoke it prior to SetBatchSize. It will be a workaround, but at least we'll have _one_ place where we can change/improve batch size inference instead of spreading it across operators. It also allows for some smart(er) solutions or non-local batch source.